packages/vanilla/src/basics/form-elements/input-button/input-button.js

```diff
@@ -106,6 +117,10 @@ class InputButton extends Core {
     return this._findDOMEl('.' + this.inputClass, this.el);
   }
 
+  _wrapperEl(){
+    return this._findDOMEl('.hig__input-button__input-wrapper', this.el);
+  }
+
   _checkedClass() {
     return `${this.wrapperClass}--checked`;
```

Comment:
Consider removing this if we don't style against this class. Should be able to style against `:checked`. Same for `:disabled`